.github/workflows/ci.yml

```diff
@@ -36,3 +36,14 @@ jobs:
           node-version: "14"
       - run: yarn
       - run: yarn test
+  lighthouse:
+    runs-on: ubuntu-latest
+    needs: build
+    name: Lighthouse
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "16"
+      - run: yarn
+      - run: yarn lighthouse
```

Comment:
if this is testing the local build, should it use `yarn lighthouse:local`?

Reply:
`local` requires a chrome instance to be installed and afaik GHA does not have that - so the alternative is to test the remote, UAT url

Reply:
This action runs on pushes to main. `ci.yml` does not actually deploy the react app, from what I can see. This lighthouse test would test the previous deployment, not the current one. For this to work, it either needs to test the local build (perhaps by installing a chromium instance in the GHA) or deploy the app **before** lighthouse testing it. The latter could be tricky, as there is no guarantee that deploying the new app will instantly make it appear at the UAT url (could take a minute or two).

Reply:
Added a local chrome instance to `ci.yml` which should allow local testing for the GHA